### PR TITLE
Labours: Package refactor + type hints + reformat

### PR DESCRIPTION
**Pull request by akx**
As discussed in #314's comments, this PR refactors the monolithic `labours.py` file into multiple files.

I also ran Monkeytype to add some type hints and Black to reformat some long lines.

A future improvement would be to get rid of `args` being passed around as-is.

### CODE REVIEW

File: python/labours/burndown.py [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    try:
+        from pandas.plotting import register_matplotlib_converters
+
+        register_matplotlib_converters()
+    except ImportError:
+        pass
+    return pandas
+
+
+def load_burndown(
```
Comment (vmarkovtsev) [inline python/labours/burndown.py]: All the modules inside `modes` contain `load_*` functions inside, so I guess we should move this one to `modes/burndown.py` too?

File: python/labours/pb_pb2.py [hunk 2]
```diff
@@ -2,11 +2,13 @@
 # source: pb.proto
 
 import sys
-_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
+
```
Comment (vmarkovtsev) [inline python/labours/pb_pb2.py]: We can discard this change

File: python/labours/plot.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import os
```
Comment (vmarkovtsev) [inline python/labours/plot.py]: How about `plotting.py`

File: python/labours/web_server.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import threading
```
Comment (vmarkovtsev) [inline python/labours/web_server.py]: We should name it `cors_web_server.py`

File: python/labours/burndown.py (hunk 1 above)
Comment (akx) [reply python/labours/burndown.py]: The idea here was that these are (slightly) lower-level burndown-processing functions, while the ones in `modes` are higher-level entry points. 

I suppose the differentiator is that these functions don't use `args` as an argument, like the modes do :)

File: python/labours/pb_pb2.py (hunk 2 above)
Comment (akx) [reply python/labours/pb_pb2.py]: Ah yeah, probably an automated change made by PyCharm's refactoring bits. Sorry for the noise.

File: python/labours/plot.py (hunk 3 above)
Comment (akx) [reply python/labours/plot.py]: Sounds good to me!

File: python/labours/web_server.py (hunk 4 above)
Comment (akx) [reply python/labours/web_server.py]: Done.

File: python/labours/burndown.py (hunk 1 above)
Comment (vmarkovtsev) [reply python/labours/burndown.py]: Well, for example, `load_devs_parallel` or `load_overwrites_matrix` do not use `args`, too. Let's move `load_burndown` to `modes` for the sense of uniformness. The rest of the stuff is indeed not "frontendish".

Comment (akx) [reply python/labours/burndown.py]: Okiedoke! Moved them in a subsequent commit.

**Pull request merged**